agent/agent-tooling/src/main/java/com/microsoft/applicationinsights/agent/internal/telemetry/TelemetryClient.java

```diff
@@ -236,17 +236,15 @@ private BatchItemProcessor initBatchItemProcessor(
     if (tempDir == null) {
       telemetryPipelineListener =
           new DiagnosticTelemetryPipelineListener(
-              "Sending telemetry to the ingestion service", false, " (telemetry will be lost)");
+              "Sending telemetry to the ingestion service", true, " (telemetry will be lost)");
     } else {
       telemetryPipelineListener =
           TelemetryPipelineListener.composite(
               // suppress warnings on retryable failures, in order to reduce sporadic/annoying
```

Comment:
please update the comment.. this is not what it's actually doing

Reply:
I think the comment is correct, can you double check?